gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.5.1-bin.zip
```

Comment:
gradle wrapper is currently at version '6.1.1', can be updated with
```
gradlew wrapper --version 6.1.1
```
in terminal

Reply:
the command is actually 
```
gradlew wrapper --gradle-version 6.1.1
```